app/src/main/java/com/google/homesampleapp/chip/ChipClient.kt

```diff
@@ -236,4 +236,65 @@ class ChipClient @Inject constructor(@ApplicationContext context: Context) {
     Timber.d("getDiscoveredDevice(${index})")
     return chipDeviceController.getDiscoveredDevice(index)
   }
+
```

Comment:
Add comment for new logical section. E.g.
  // ---------------------------------------------------------------------------
  // Access clusters via numeric ids. Useful to access manufacturer specific clusters.
  // See https://github.com/google-home/sample-app-for-matter-android/issues/76.